acquisitions/src/main/resources/thunderjet/consortia/order-utils/inventory-college.feature

```diff
@@ -0,0 +1,268 @@
+Feature: global inventory
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(collegeTenant)', 'Accept': 'application/json' }
+
+  Scenario: create identifier types
```

Comment:
how it worked before in other order releated karate tests, was some code already to create identifier types, instance types, loan types etc?
Also scenario - it is some completed flow like create data and perform test, just creating identifier types can not be considered as scenario

Reply:
The files in `order-utils` are copied from `global`. Difference is that in global headers are defined to work for diku. I think that files should not contain scenarios as well and should contain functions with parameters to be able to reuse them.
I think better examples are functions `createFund`, `createFiscalYear`, `createLedger`, etc. They are read from `*/reusable` packages and defined in `karate-config.js`.